internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanager/scanners/qualys/handler/QualysScanHandler.java

```diff
@@ -0,0 +1,604 @@
+import com.jcraft.jsch.SftpException;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.log4j.Logger;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+import org.wso2.security.tools.scanmanager.common.model.LogType;
+import org.wso2.security.tools.scanmanager.common.model.ScanStatus;
+import org.wso2.security.tools.scanmanager.scanners.common.ScannerConstants;
+import org.wso2.security.tools.scanmanager.scanners.common.exception.ScannerException;
+import org.wso2.security.tools.scanmanager.scanners.common.util.CallbackUtil;
+import org.wso2.security.tools.scanmanager.scanners.common.util.ErrorProcessingUtil;
+import org.wso2.security.tools.scanmanager.scanners.common.util.FileUtil;
+import org.wso2.security.tools.scanmanager.scanners.qualys.QualysScannerConstants;
+import org.wso2.security.tools.scanmanager.scanners.qualys.config.QualysScannerConfiguration;
+import org.wso2.security.tools.scanmanager.scanners.qualys.model.ScanContext;
+import org.wso2.security.tools.scanmanager.scanners.qualys.util.RequestBodyBuilder;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.StringReader;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerException;
+
+/**
+ * This class is responsible to handle the required  use cases of Qualys scanner.
+ */
+public class QualysScanHandler {
+
+    private static final Logger log = Logger.getLogger(QualysScanHandler.class);
+
+    //    private final Log log = LogFactory.getLog(QualysScanHandler.class);
+    private QualysApiInvoker qualysApiInvoker;
+
+    public QualysScanHandler(QualysApiInvoker qualysApiInvoker) {
+        this.qualysApiInvoker = qualysApiInvoker;
+    }
+
+    public QualysApiInvoker getQualysApiInvoker() {
+        return qualysApiInvoker;
+    }
+
+    /**
+     * Prepare the scan before launching the scan. Main tasks are Adding the authentication scripts and crawling
+     * scripts.
+     *
+     * @param fileMap Map that contains the file paths.
+     * @param appID   Application ID
+     * @param jobId   Job ID
+     * @param appName Web Application Name
+     * @param host    host url of qualys
+     * @return Authentication script id
+     * @throws ScannerException Error occurred while adding authentication scripts
+     */
+    public String prepareScan(String appID, String jobId, String appName, Map<String, List<String>> fileMap,
+            String host) throws ScannerException {
+        String authScriptId;
+        //Download authentication script form ftp location.
```

Comment:
Keep one line before one line comment. Do the same for  similar other places.
```suggestion

        //Download authentication script form ftp location.
```